upf/fnode.py

```diff
@@ -37,18 +37,15 @@ def __init__(self, content: FNodeContent, node_id: int):
     # issue since, by default, equality coincides with identity.
     def __hash__(self) -> int:
         return self._node_id
-
+    
```

Comment:
Remove these trailing whitespaces.

To prevent them you can enable an option in your vscode (https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut).

Reply:
Thank you very much for the tip.